lib/annotate_rb/options.rb

```diff
@@ -76,7 +76,8 @@ def from(options = {}, state = {})
       target_action: :do_annotations, # Core; Possible values: :do_annotations, :remove_annotations
       wrapper: nil, # ModelAnnotator, RouteAnnotator
       wrapper_close: nil, # ModelAnnotator, RouteAnnotator
-      wrapper_open: nil # ModelAnnotator, RouteAnnotator
+      wrapper_open: nil, # ModelAnnotator, RouteAnnotator,
+      classes_default_to_s: [] # ModelAnnotator
     }.freeze
 
```

Comment:
We'll also want to add the configuration key name to `OTHER_OPTION_KEYS` so that it gets picked up in default options.

Reply:
Done